src/types/base.ts

```diff
@@ -23,26 +23,37 @@ type MyInfoSourceDefault = Exclude<MyInfoSource, MyInfoSource.NotApplicable>
 // For a full reference, see https://www.ndi-api.gov.sg/library/myinfo/implementation-myinfo-data
 export type MyInfoNotApplicable = SourceProp<MyInfoSource.NotApplicable>
 
-export type MyInfoApplicable<S> = {
-  lastupdated: string
+export type MyInfoApplicable<S extends MyInfoSource> = {
+  // The “lastupdated” field is not always non-empty currently, but under what circumstances this occurs is unknown
+  lastupdated?: string
   classification: MyInfoDataClassification.Confidential
 } & SourceProp<S>
 
 type UnavailableProp<T> = {
   unavailable: T
```

Comment:
if the value of unavailable is always false, then should we not just type it here as `false`?